config/config.txt

```diff
@@ -297,7 +297,7 @@ ANNOUNCE_ADMIN_LOGOUT
 
 ## AUTOADMIN
 ## Uncomment to automatically give that admin rank to all players
-#AUTOADMIN Game Admin
+AUTOADMIN Game Admin
```

Comment:
We don't use the config on here but maybe this isn't a great idea to have as default